messaging/rabbitmq_policy.py

```diff
@@ -107,13 +111,23 @@ def list(self):
         policies = self._exec(['list_policies'], True)
 
         for policy in policies:
-            policy_name = policy.split('\t')[1]
-            if policy_name == self._name:
+            policy_data = policy.split('\t')
+
+            policy_name = policy_data[1]
+            apply_to = policy_data[2]
+            pattern = policy_data[3].replace('\\\\', '\\')
```

Comment:
@romanek-adam, `r'\'` is not a valid string literal in python. I can change this line only like this:

```python
pattern = policy_data[3].replace(r'\\', '\\')
```

I don't think that is better idea to use different strings in this place. In my point of view it can be confusing. I suggest to keep just stings here instead of mix of raw strings and strings here. What do you think?